src/main/react-front-end/src/views/Transfer/TransferComponent.js

```diff
@@ -49,7 +50,7 @@ export default class TransferComponent extends Component {
         compress: "true",
         retry: 5
       },
-      compact: true
+      compact: cookies.get('compact') == 'true'
```

Comment:
Please use redux store for coherence.

Reply:
Please update all occurrences to store and access the view type from redux store. Using keyword 'compactViewEnabled' is preferred.